File: BAEKJOON/string/2941.py
# 2941: 크로아티아 알파벳/ 실버 5

cro_alpha = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z=']

result = 0
data = input()

for i in cro_alpha:
    result += data.count(i)
    data = data.replace(i, ' ')

result += sum(map(lambda x: len(x), data.split()))
print(result)

File: BAEKJOON/string/javascript/2941.js
/*
 2941: 크로아티아 알파벳/실버 5
*/
const fs = require('fs');
const localPath = __dirname + '/input.txt';  // 로컬 실행용 경로
const baekPath = '/dev/stdin';  // 백준 제출용 경로
let input = fs.readFileSync(localPath).toString().trim();

const croAlpha = ['c=', 'c-', 'dz=', 'd-', 'lj', 'nj', 's=', 'z='];

let result = input;
for (const char of croAlpha) {
  result = result.replace(new RegExp(`${char}`, 'ig'), '*');
}

console.log(result.length);